### PR TITLE
Classifier App for winML

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

File: apps/mivisonx_winml_classifier/classifier.cpp [hunk 1]
```diff
@@ -0,0 +1,1225 @@
...
+#define strncasecmp _strnicmp
+#define strcasecmp _stricmp
+#endif
+
+using namespace cv;
+using namespace std;
+
+
+#if ENABLE_OPENCV
+#include <opencv2/opencv.hpp>
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/classifier.cpp]: Defined in line 25

File: apps/mivisonx_winml_classifier/classifier.cpp [hunk 2]
```diff
@@ -0,0 +1,1225 @@
...
+#include <opencv/cv.h>
+#include <opencv/highgui.h>
+#define CVUI_IMPLEMENTATION
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/classifier.cpp]: redefined

File: apps/mivisonx_winml_classifier/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Image Clasiification using AMD WinML Extension
+
+This project shows how to run supported classification models with MIVisionX RunTime:
+
+* A c/c++ implementation for image classification for live camera capture.
+
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/README.md]: Add a few screenshots from the app

File: apps/mivisonx_winml_classifier/classifier.cpp [hunk 4]
```diff
@@ -0,0 +1,1215 @@
+#if _WIN32
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/classifier.cpp]: This is a windows app, no need to check.

File: apps/mivisonx_winml_classifier/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Image Clasiification using AMD WinML Extension
+
+This project shows how to run supported classification models with MIVisionX RunTime:
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/README.md]: We need to reword this. This application ... and supports ... ONNX models.

File: apps/mivisonx_winml_classifier/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Image Clasiification using AMD WinML Extension
+
+This application shows how to run supported ONNX models with MIVisionX RunTime:
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/README.md]: This application implements a method to run WinML supported ONNX models using MIVisionX RunTime. This c/c++ implementation does image classification on live camera capture or pre-recorded video stream. 

This application is a sample for developing windows application using MIVisionX WinML extension.

File: apps/mivisonx_winml_classifier/README.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  * Add `OpenCV_DIR\x64\vc14\bin` or `OpenCV_DIR\x64\vc15\bin` to your `PATH`
+
+### Build using `Visual Studio 2017` on 64-bit Windows 10
+* Use `apps/mivisonx_winml_classifier/winml_classifier.sln` to build for x64 platform
+
+## Run
+
+### Step 1. Get ONNX model
+* Train your own YoloV2 ONNX model or get it from [onnx github](https://github.com/onnx/models). ONNX version 1.3 is recommended.
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/README.md]: Yolo not supported in this app? WinML supported Classification ONNX models.

**Comment by kiritigowda** [comment]
@hansely review and test this application.

File: apps/mivisonx_winml_classifier/README.md (hunk 6 above)
Comment (kiritigowda) [reply apps/mivisonx_winml_classifier/README.md]: resolve this

File: apps/mivisonx_winml_classifier/README.md (hunk 7 above)
Comment (kiritigowda) [reply apps/mivisonx_winml_classifier/README.md]: resolve this

File: apps/mivisonx_winml_classifier/classifier.cpp [hunk 8]
```diff
@@ -0,0 +1,1238 @@
...
+
+	//input tensor dimensions
+	vx_size dims_data_224X224[4] = { 224, 224, 3, 1 };
+
+	// output tensor prob dimensions
+	vx_size dims_prob[2] = { 1000, 1 };
+	vx_size dims_prob_2[4] = { 1, 1, 1000, 1 };
+
+	char labelBuf[2048];
+	//labelFileName = "C:\\Users\\lakshmi\\work\\MIVisionX_winmlFix\\MIVisionX_winmlFix\\amd_openvx_extensions\\amd_winml\\samples\\data\\Labels.txt";
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/classifier.cpp]: remove

File: apps/mivisonx_winml_classifier/README.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+ --vgg19      <FULL_PATH_TO\vgg19\model.onnx>        [OPTIONAL] 
+ --shufflenet <FULL_PATH_TO\shufflenet\model.onnx>   [OPTIONAL] 
+ --squeezenet <FULL_PATH_TO\squeezenet\model.onnx>   [OPTIONAL] 
+ --densenet   <FULL_PATH_TO\densenet\model.onnx>     [OPTIONAL] 
+ --zfnet      <FULL_PATH_TO\zfnet\model.onnx>        [OPTIONAL] 
+ --label FULL_PATH_TO\amd_openvx_extensions\amd_winml\samples\data\Labels.txt [REQUIRED]
+ --capture 0  [REQUIRED]
+```
+## MIVisionX Image Classification
+![MIVisionX Image Classification](images/MIVisionX-ImageClassification.png)
```
Comment (kiritigowda) [inline apps/mivisonx_winml_classifier/README.md]: change img

File: apps/mivisonx_winml_classifier/classifier.cpp [hunk 10]
```diff
@@ -0,0 +1,1432 @@
...
+#include <fstream>
+#include <opencv2/opencv.hpp>
+#include <opencv/cv.h>
+#include <opencv/highgui.h>
+#include <algorithm>
+#define CVUI_IMPLEMENTATION
+#include "cvui.h"
+
+
+#ifdef _MSC_VER
```
[the remark this answers is not in the document]
Comment (LakshmiKumar23) [reply apps/mivisonx_winml_classifier/classifier.cpp]: doesn't build if I remove this

**Pull request merged**